### DISCUSSION TITLE
no title

### DISCUSSION THREAD

**Discussion by catmeowjiao** [ITCraftDevelopmentTeam/Forum#46] [General]
我说清楚前两天那件事发生时我的内心想法.
1.我泄露校名并撤回后, xxtg通过防撤回看到了我的校名并发了出来, 我认为暴露校名和暴露姓名或住址是同一级别的, 所以我就骂了;
2.在我骂了之后, xxtg发了群规里的"友好交流", 我认为这是泄露他人隐私还不让人骂, 就像退群清净清净;
3.xxtg用tg给我说了一些话, 详见截图;
4.第二天我为了重新入群, 写了很多, 然后投票结果不通过, 我很生气, 就骂了;
5.被驳回的issue中, chun-awa发送"深表nm歉意", "我们对宁加入申请未通过不表歉意", 我认为是阻止他人保护自己的隐私.
就说这些.
![image](https://github.com/ITCraftDevelopmentTeam/Forum/assets/138079152/8042d251-d840-475b-ab72-3243d7449853)
![image](https://github.com/ITCraftDevelopmentTeam/Forum/assets/138079152/573c5b8a-17aa-4fa0-bf84-b94493a662db)
![image](https://github.com/ITCraftDevelopmentTeam/Forum/assets/138079152/7f8fd93e-303f-446b-b0fb-8da5bb743c52)

**Comment by This-is-XiaoDeng** [comment]
晚上好

对于您的加入申请在两次投票中均被驳回，我们深表遗憾。

关于第五点的内容，我们已核实这部分信息并对 @chun-awa 进行了教育；但是我们不认为 “我们对宁加入申请未通过不标歉意” 等发言与您保护隐私有任何联系，请您不要过度理解。

保护隐私是有必要的，也是我们每个人的责任。但是“保护隐私”与“友好交流”并不冲突，并不是一定要用“问候家人”的方式保护所谓的隐私。我们建议您在以后的交流中保持理智。

祝您身体健康。

**Comment by catmeowjiao** [comment]
请友好交流（x

**Reply by chun-awa** [reply]
所以你到底要干啥，是让xxtg出来解释吗，要是是这样的话就没我事了（

**Reply by catmeowjiao** [reply]
no
声明，避免有人说我自找的

**Reply by chun-awa** [reply]
那没事了

**Reply by catmeowjiao** [reply]
那没事了=nmsl=你妈死了（确信

**Reply by This-is-XiaoDeng** [reply]
我没骂你很给你面子了（

**Comment by catmeowjiao** [comment]
遗言：在这个浮躁的年代，开放包容、理性客观的思考可能不是最打动人心的，但一定是人们最需要的。

**Comment by xxtg666** [comment]
byd遗言是吧，我的打算不是说拒绝你再回来，是想等你小学毕业后上了初中学几年再回来，毕竟这次事情还是能看出来你处理问题的方式太xxs

**Comment by DQB061204** [comment]
对于您的申请 我对泄露您隐私这件事情深表歉意 但是您也不能通过辱骂等方式对我们管理层人员的错误进行反击 我认为 您和xxtg之间应该互相承认错误 并且要对chun-awa对于您的辱骂表示强烈的批评 我们将完善群规 并且使用更加文明的语言进行协商
综上所述 我决定 暂停您的加入申请十五天 十五天后您再申请加入 谢谢！
